app/controllers/SavedDeclarationsController.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import connectors.CustomsDeclareExportsConnector
+import controllers.actions.AuthAction
+import javax.inject.Inject
+import models.requests.ExportsSessionKeys
+import models.{Mode, Page}
+import play.api.Logger
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.saved_declarations
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SavedDeclarationsController @Inject()(
+  authenticate: AuthAction,
+  customsDeclareExportsConnector: CustomsDeclareExportsConnector,
+  mcc: MessagesControllerComponents,
+  savedDeclarationsPage: saved_declarations
+)(implicit ec: ExecutionContext)
+    extends FrontendController(mcc) with I18nSupport {
+
+  private val logger = Logger(this.getClass)
+  private val defaultPage = Page()
```

Comment:
This will need to change when implementing pagination (sep. ticket).  The back-end service already supports it.    I agree that we don't particularly need the 'defaultPage' constant declaring though.  I'll remove it and put a default in the connector.

Reply:
I think it's fine where it is tbh, makes it easier to implement pagination later on having it at this level

Reply:
I maybe wouldnt use default pagination though as you ideally want `pageSize=Integer.MAX`

Reply:
The ticket calls for 50 items on page (at this point).    Testers will be bound to test that!